app/components/avatar-list.tsx

```diff
@@ -48,13 +54,14 @@ export default function AvatarList({ className = "", list, ...rest }: AvatarList
         })}
         {remaining && remaining.length > 0 && (
           <div
+            aria-hidden="true"
```

Comment:
I added a couple of this, I read about it but I still don't know if it's a good move to use them. What would you do?

Reply:
I honestly don't know what they do